src/main/scala/org/apache/spark/sql/OapTestSuite.scala

```diff
@@ -113,6 +116,13 @@ abstract class OapTestSuite extends BenchmarkConfigSelector with OapPerfSuiteCon
   def resultMap = _resultMap
 
   protected def testSet: Seq[OapBenchmarkTest]
+  protected def dropCache(): Unit = {
+    val nodes = spark.sparkContext.getExecutorMemoryStatus.map(_._1.split(":")(0))
+    nodes.foreach { node =>
+      val dropCacheResult = Seq("bash", "-c", s"""ssh $node "echo 3 > /proc/sys/vm/drop_caches"""").!
```

Comment:
Does it have privilege issue?? Can sudo work remotely??

Reply:
We run the script in driver with sudo or root (effectively runs as super user), and ssh sure can work remotely.